src/Compilers/Test/Utilities/CSharp/BasicCompilationUtils.cs

```diff
@@ -37,7 +37,9 @@ private static VisualBasicCompilation CreateCompilationWithMscorlib(string sourc
             return VisualBasicCompilation.Create(assemblyName, new[] { tree }, references, options);
         }
 
-        private static BasicTestBase s_instance = new BasicTestBase();
+        private static BasicTestBase s_instance;
```

Comment:
Here is the reasoning from the commit message:

> Several of our test suites have cycles between static field
initializers and the containing types constructor.  The mono runtime
ends up stack overflowing on these cycles and prevents us from running
suites.  Changing the suites to remove this cycle via lazy field
initialization.

Essentially Mono can't handle the cycles we have here between instance and static ctors and ends up stack overflowing.  Have to do this to break the cycle so we can run our tests on Mono.